examples/ccvm_boxqp_plot.py

```diff
@@ -50,10 +50,17 @@
     # Save metadata to file
     metadata_filepath = metadata_list.save_metadata_to_file(METADATA_DIR)
 
+    # Customize machine_parameters
+    machine_parameters = {
+        "cpu_power": {20: 5.0, 30: 5.0, 40: 5.0, 50: 5.0, 60: 5.0, 70: 5.0}
+    }
+
     plot_fig, plot_ax = ccvmplotlib.plot_TTS(
         metadata_filepath=metadata_filepath,
         problem="BoxQP",
-        TTS_type="wallclock",
+        machine_time_func=solver.cpu_machine_time(
```

Comment:
I believe this won't run until we have the solver bits written, too, and we don't want to merge something that can't be run. However, I like that this PR is staying within scope, thank you.
My recommendation to merge this fast is to add an example of how the user can define their own cpu_machine_time() function _in this file_. As long as this file runs, the exact implementation of that function doesn't matter, but for the sake of making the best code we can let's use the cpu implementation.